wca-regulations.md

```diff
@@ -202,6 +202,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
         - 7f1d) Surface: The flat surface on which the Stackmat has been placed. The mat is considered a part of the surface. The timer is not considered a part of the surface.
     - 7f2) The Stackmat timer must be attached to the mat and placed on the surface, with the timer on the side of the mat nearest to the competitor.
         - 7f2a) Exception: For 3x3x3 With Feet, the Stackmat must be placed directly on the floor. The timer device may be placed on the side of the mat farthest from the competitor.
+    - 7f3) Organizers should strongly consider modifications to make Stackmat timers more robust against common incidents, such as making the buttons more difficult to press by accident (e.g. by attaching O-rings around the buttons) and securing the battery more firmly (e.g. by padding the battery compartment).
```

Comment:
Just realized this is one of the only two `Organizers` in the whole Regulations and Guidelines.
Should we change to `The organization team` so the terminology is the same throughout?

The other "organizers" is in [4b2++](https://www.worldcubeassociation.org/regulations/guidelines.html#4b2++)

Reply:
Yes, we should make format consistent. Thanks for spotting it!

Reply:
```suggestion
    - 7f3) The organization team should strongly consider modifications to make Stackmat timers more robust against common incidents, such as making the buttons more difficult to press by accident (e.g. by attaching O-rings around the buttons) and securing the battery more firmly (e.g. by padding the battery compartment).
```